chevah/empirical
# [#2453] Update OS X support.

# Problem

Some testing support code don't work on OSX
# Changes

Update the exceptions.
There were not many changes as OSX is pretty standard unix and we used to suppport OSX in the past.
Update paver.
# How to test

reviewers: @alibotean 

check that changes make sense. this is more to let you know about the changes
